Tools/cases_generator/generate_cases.py

```diff
@@ -48,6 +48,34 @@
 )
 
 
+def effect_size(effect: StackEffect) -> tuple[int, str]:
```

Comment:
Ah, I get this now (after looking at the tests). Maybe just add a docstring here (and maybe to the functions below) saying something to the effect of "the first integer value is the constant, statically-analyzable stack effect part, and the second string value is the symbolic, not-statically-analyzable stack effect part.